packages/editor-ui/src/components/Node.vue

```diff
@@ -161,6 +180,42 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.setSubtitle();
 			}
 		},
+		nodePosition: {
+			handler() {
+				if (this.showWebhookNodeTooltip) {
+					this.showWebhookNodeTooltip = false;
+					setTimeout(() => {
+						this.showWebhookNodeTooltip = true;
+					}, 200);
+				}
+			},
+			immediate: true,
+			deep: true,
+		},
+		nodeOffsetPosition: {
+			handler() {
+				if (this.showWebhookNodeTooltip) {
+					this.showWebhookNodeTooltip = false;
+					setTimeout(() => {
+						this.showWebhookNodeTooltip = true;
```

Comment:
need to be careful here.. what if node finishes running after offset changes before tooltip shows again?

Reply:
I added one more thing here, to check if the workflow is running. This way we will protect from case that you mentioned.